roles/check_os/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+
+- name: 'Assertion on OS'
```

Comment:
Is there a way to 'override' this assertion?

Reply:
Done